chapter05/chapter05.rpy

```diff
@@ -137,7 +137,7 @@ label c05_intro:
     I stared at the now-embarrassed Celia, trying to figure out her motives.
     Under my gaze, Celia's eyes wander the room, a soft blush forming on her cheeks.
     ... [Pause] Cute.
-    Looking down at the sketchbook, I carefully ripped a single sheet of paper along with the holed-rom, making the paper clean.
```

Comment:
Well, the type of sketchbook I had in mind was a black sketchbook that was fairly large with spirals on the side- sort of like a normal notebook. There are other sketchbooks where it's sewn together like a composition notebook, and also has the extra side for people to pull the real page out. They're common over where I'm at, so I wrote it like that.